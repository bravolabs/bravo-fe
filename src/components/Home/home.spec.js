import React from 'react';
import renderer from 'react-test-renderer';

import Home from './index';

describe('Home', () => {
  it('snapshot renders', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
