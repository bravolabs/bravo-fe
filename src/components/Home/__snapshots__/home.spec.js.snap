// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot renders 1`] = `
<h1
  className="sc-bdVaJa lpjCXg"
>
  Welcome to Bravo-FE
</h1>
`;
